### PR TITLE
Print oereb ju

### PR DESCRIPTION
**Pull request by voisardf**
This PR is to apply in combination with PR: https://github.com/sitn/crdppf/pull/317 as it implements a switch in the core to choose the preferred render engine for the PDF extract.

It adds the parameter to the CONST_vars and handles the switch in the portal interface too.

### CODE REVIEW

File: crdppf/static/js/Crdppf/main.js [hunk 1]
```diff
@@ -295,13 +295,21 @@ Crdppf.init_main = function(lang) {
...
+                              Crdppf.currentProperty.attributes.id
+                          ].join('');
+                        } else {
+                          var url = [
+                              Crdppf.printReportCreateUrl,
+                              selectedRadio.inputValue,
+                              "/pdf/",
+                              Crdppf.currentProperty.attributes.egrid
+                          ].join('');
+                        }
```
Comment (maltaesousa) [inline crdppf/static/js/Crdppf/main.js:312]: This is ok and it works, but a good practise here would be to not repeat yourself (DRY). You could, for instance do something like that:
```javascript
var printParams = {}
if(Crdppf.pdfRenderEngine == 'crdppf_mfp') {
  printParams = {
    url: "/",
    id: "id"
  }
} else {
  printParams = {
    url: "/pdf/",
    id: "egrid"
  }
}

var url = [
  Crdppf.printReportCreateUrl,
  selectedRadio.inputValue,
  printParams.url,
  Crdppf.currentProperty.attributes[printParams.id]
].join('');
```
It's easier to understand what changes when `crdppf_mfp` mode is on of off...

Another good way to deal with this is to put that in the settings. In your .yaml:
```yaml
pdf_renderer: overwrite_me

print_params:
  crdppf_mfp:
    url: /
    id: id
...
```
And then access it directly in your .js:

```javascript
var url = [
  Crdppf.printReportCreateUrl,
  selectedRadio.inputValue,
  settings['pdf_render_engine']['url']
  settings['pdf_render_engine']['id']
].join('');
```

Comment (voisardf) [reply crdppf/static/js/Crdppf/main.js:312]: thanks a lot. Very good suggestions! 👍

**Comment by sitj-raphael** [comment]
Fonctionne très bien chez nous

**Comment by voisardf** [comment]
cool. Je prévois d'améliorer le code selon la deuxième suggestion de Stéphane, car cela généralise mieux.
Stay tuned ;)

**Comment by voisardf** [comment]
Going with this version for a start. merging

**Pull request merged**